virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -38,6 +38,26 @@ def get_non_root_disk_name(session):
     return get_non_root_disk_names(session)[0]
 
 
+# pylint: disable=C0402
+def is_image_mode(session):
```

Comment:
If someone really needs this function, I suggest we put it to utils_sys.py as this is not only used by libvirt disk test.
We can put this function in a seperated PR.

Reply:
@hholoubk https://github.com/avocado-framework/avocado-vt/pull/4215 FYI.